runtime/nls/vrfy/j9bcv.nls

```diff
@@ -406,6 +406,20 @@ J9NLS_BCV_ERR_WRONG_INIT_METHOD.system_action=The JVM will throw a verification
 J9NLS_BCV_ERR_WRONG_INIT_METHOD.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
 
+J9NLS_BCV_SUPERCLASS_IS_VALUE_TYPE=bad class %s: superclass is value type class
```

Comment:
I can add the length formatting directly like so - `%.*s`, but I can't use the numbered arguments like the examples in the file because `setCurrentExceptionNLSWithArgs` 
does not perform the argument replacement. I can either: 
 - do it directly like I mentioned
 - change `setCurrentExceptionNLSWithArgs` (which doesn't seem to be an issue, as I can't find it being used anywhere else)
 - or abandon the class name printing altogether.

Reply:
I don't want to change a bunch of files for this small and specific change, so if it isn't acceptable to just stick a `%.*s` in there then it's probably not worth it - especially on this branch

Reply:
I believe they have to be re-ordered because the translations may move around the names. The length is needed because our strings are not always null terminated.

> I can't use the numbered arguments like the examples in the file because setCurrentExceptionNLSWithArgs
does not perform the argument replacement

`setCurrentExceptionNLSWithArgs` calls `j9str_vprintf` which I believe does the arg reordering. So it should just work without modification. What issues were you encountering?

Also, I just noticed that these nls messages need to be in j9vm.nls

Reply:
Oh ok, I didn't quite understand where the reordering was done, thanks for the clarification.